Vagrantfile

```diff
@@ -1,14 +1,16 @@
 Vagrant.configure(2) do |config|
+  config.vm.synced_folder ".", "/mk"
```

Comment:
Afaik, Vagrant already synces this directory as `/vagrant`. Would this cause `.` to appear twice or would this just cause `.` to appear once as `/mk`. In the latter case, I don't have any objections to this change (I actually find it an improvement).